log/log.go

```diff
@@ -46,3 +46,9 @@ func Trace(msg string, ctx ...interface{}) {
 	metrics.GetOrRegisterCounter("trace", nil).Inc(1)
 	l.Output(msg, l.LvlTrace, CallDepth, ctx...)
 }
+
+// GetHandler return the Handler assigned to root
+func GetHandler() l.Handler {
+	lh := l.Root()
```

Comment:
`lh` is a a `Logger`-typed variable. therefore, it should be named something other than this

Reply:
I think it's not needed.
I replaced it directly with 

> return l.Root().GetHandler()

Reply:
even better :+1: